ui/app/rif/components/searchDomains.js

```diff
@@ -2,16 +2,34 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import rifActions from '../../rif/actions'
+import actions from '../../actions'
 
 class SearchDomains extends Component {
 	_handleKeyDown = (e) => {
 		if (e.key === 'Enter') {
+			let insertedDomain = e.target.value.toLowerCase()
+			//Theres a limitation in manager that domains with less 5 characters are blocked
+			if(insertedDomain.length <= 5){
+				this.props.displayWarning("Domains with less than 5 characters are blocked.")
+				return
+			}
+
 			let domains = JSON.parse(localStorage.rnsDomains);
-			let existDomain = domains.find(domain => domain.domain === e.target.value.toLowerCase())
+			let existDomain = domains.find(domain => domain.domain === insertedDomain)
 			if(existDomain)
 				return this.props.showDomainsDetailPage(existDomain)
-
-			//Here goes the logic to search domains that are not in localstorage
+			//Checks if the domain is available, so if it is, it need to render a screen so the user can register it
+			this.props.checkDomainAvailable(insertedDomain).then(domain => {
+				if(domain.length > 0)
+					this.props.showDomainRegisterPage(domain)
+				else{
+					this.props.getDomainDetails(insertedDomain).then(ret => {
+						console.log("I finished getDomainDetail", ret)
```

Comment:
Same here, lets use console.debug

Reply:
Changed all console.log to .debug, thnks